Add .env.example file and update requirements for computer-use

Zochory opened the pull request:
This pull request introduces updates to the configuration and dependencies for the `computer-use` project. The changes include adding new environment variables for API keys and updating the `requirements.txt` file with specific versions for several dependencies.

Configuration updates:

* [`computer-use/.env.example`](diffhunk://#diff-793ec455a5279ef64b323deed01125f2349adbdfdcc24a9270b2109f87c5b793R1-R10): Added environment variables for Azure OpenAI and OpenAI API keys.

Dependency updates:

* [`computer-use/requirements.txt`](diffhunk://#diff-42fecd89850abd77db2aaa3be1dfc0ac4183c6c1f9302a23147a5b61c1a5fc20L2-R8): Updated versions for `pyautogui` and `requests`, and added new dependencies: `Pillow`, `python-dotenv`, `pynput`, `mss`, and `numpy`.## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
* ...

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[ x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[x ] Bugfix
[ ] Feature
[ x ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone zochory/computer-use-model
cd computer-use-model
git checkout add-env-and-gitignore
npm install
```

* Test the code
<!-- Add steps to run the tests suite and/or manually test -->
```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
<!-- Add any other helpful information that may be needed here. -->

Zochory commented:
@microsoft-github-policy-service agree [company="Qredence"]

Zochory commented:
@microsoft-github-policy-service agree

lutzroeder commented:
Closing this pull request due to inactivity. If you'd like to continue, please comment or reopen.

Zochory commented:
Is ok,  didn't went to update it et maintain it , and for those that might want to see why backthen I added those : 

This pull request introduces environment variable configuration for API keys and updates the dependencies in `requirements.txt` to include new libraries and specify version constraints. These changes aim to enhance configuration flexibility and ensure compatibility with the required libraries.

### Configuration Updates:
* Added placeholders for `AZURE_OPENAI_ENDPOINT`, `AZURE_OPENAI_API_KEY`, and `OPENAI_API_KEY` in the `.env.example` file to support both Azure OpenAI and OpenAI API configurations. (`computer-use/.env.example`, [computer-use/.env.exampleR1-R10](diffhunk://#diff-793ec455a5279ef64b323deed01125f2349adbdfdcc24a9270b2109f87c5b793R1-R10))

### Dependency Updates:
* Updated `requirements.txt`:
  - Upgraded `pyautogui` and `requests` to specific minimum versions.
  - Added new dependencies: `Pillow` for image processing, `python-dotenv` for environment variable management, `pynput` for input device control, `mss` for screenshot functionality, and `numpy` for numerical operations. (`computer-use/requirements.txt`, [computer-use/requirements.txtL2-R8](diffhunk://#diff-42fecd89850abd77db2aaa3be1dfc0ac4183c6c1f9302a23147a5b61c1a5fc20L2-R8))
